Plugins/Public/playercntl_plugin/Rename.cpp

```diff
@@ -455,6 +463,26 @@ namespace Rename
 
 				// The rename worked. Log it and save the rename time.
 				AddLog("NOTICE: User rename %s to %s (%s)", wstos(o.wscCharname).c_str(), wstos(o.wscNewCharname).c_str(), wstos(HkGetAccountID(acc)).c_str());
+
+				//resetrep cooldown
+				wstring wscCharFileName2;
+				HkGetCharFileName(o.wscNewCharname, wscCharFileName2);
+				wstring wscDir;	
+				HK_ERROR err;
+				if ((err = HkGetAccountDirName(o.wscNewCharname, wscDir)) != HKE_OK)
+				{
+					ConPrint(L"ERR 461 " + HkErrGetText(err));
```

Comment:
okay we could do something better than 461 here if it's printing errors to console - like the function name and that it's an HkGetAccountDirName check for resetrep. But also what about logging to disk?

Reply:
Uh, they aren't ever should happen at all. So no need to log